components/arm/fake/fake.go

```diff
@@ -0,0 +1,194 @@
+// Package fake implements a fake arm.
+package fake
+
+import (
+	"context"
+	// for arm model.
+	_ "embed"
+
+	"github.com/edaniels/golog"
+	"github.com/pkg/errors"
+	commonpb "go.viam.com/api/common/v1"
+	pb "go.viam.com/api/component/arm/v1"
+
+	"go.viam.com/rdk/components/arm"
+	"go.viam.com/rdk/components/arm/eva"
+	ur "go.viam.com/rdk/components/arm/universalrobots"
+	"go.viam.com/rdk/components/arm/xarm"
+	"go.viam.com/rdk/components/arm/yahboom"
+	"go.viam.com/rdk/components/generic"
+	"go.viam.com/rdk/config"
+	"go.viam.com/rdk/motionplan"
+	"go.viam.com/rdk/referenceframe"
+	"go.viam.com/rdk/registry"
+)
+
+// ModelName is the string used to refer to the fake arm model.
+const ModelName = "fake"
+
+//go:embed fake_model.json
+var fakeModelJSON []byte
+
+// AttrConfig is used for converting config attributes.
+type AttrConfig struct {
+	FailNew  bool   `json:"fail_new"`
```

Comment:
I think this should also be removed, no?

Reply:
This is still being used in some tests, so I wasn't inclined to take it out.  Thoughts?